src/porepy/utils/sparse_mat.py

```diff
@@ -183,6 +183,59 @@ def stack_mat(A, B):
         A._shape = (A._shape[0] + B._shape[0], A._shape[1])
 
 
+@pp.time_logger(sections=module_sections)
+def copy(A):
```

Comment:
Why not use copy functions for sparse matrices directly?

Reply:
Scipy tends to change the ordering of A.indices. Maybe there is a better way to avoid that, but this implementation is one possibility.